src/xdocs/config_imports.xml

```diff
@@ -863,6 +863,19 @@ import android.*;
             you want to match only a certain package level instead.
           </li>
         </ul>
+        <h4 id="innerclass-notes">Notes on inner classes</h4>
+        <p>
+          Static inner classes have to be explicitly allowed when they are imported, they
+          are not automatically allowed along with their enclosing class.
+        </p>
+        <p>
+          For example, to allow importing <code>java.util.Map.Entry</code> use the following configuration:
+        </p>
+        <source>
+&lt;import-control pkg=&quot;com.puppycrawl.tools.checkstyle&quot;&gt;
+    &lt;allow class=&quot;java.util.Map.Entry&quot;/&gt;
```

Comment:
Should example also allow `java.util.Map` to show that we need 2 entries if both are used since we state earlier `are not automatically allowed along with their enclosing class.`.